packages/tests/src/nwaku.ts

```diff
@@ -440,7 +440,6 @@ export function defaultArgs(): Args {
   return {
     listenAddress: "127.0.0.1",
     nat: "none",
-    relay: true,
```

Comment:
If the argument is not present, `nwaku` starts with relay by default, that is why the relay tests still pass.
 set `relay: false` instead.